examples/light-switch-app/silabs/SiWx917/build_for_wifi_args.gni

```diff
@@ -19,3 +19,4 @@ chip_enable_openthread = false
 import("${chip_root}/src/platform/silabs/SiWx917/wifi_args.gni")
 
 chip_enable_ota_requestor = true
```

Comment:
same here, remove the build_for_wifi_args.gni and build_for_wifi_gnfile.gn and have just the .gn and args.gni